### PR TITLE
Telemetry remove

### PR DESCRIPTION
**Pull request by Wxl19980214**
## Describe your changes

Use ploomber_core.telemetry 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

### CODE REVIEW

**Comment by idomic** [comment]
@Wxl19980214 is this ready for review?
Please resolve conflicts

**Comment by idomic** [comment]
@Wxl19980214 ready for review? If so, please fix the conflict

File: setup.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def read(*names, **kwargs):
     'Jinja2',
     'pyyaml',
     'ploomber>=0.14.6',
+    'ploomber_core',
```
Comment (edublancas) [inline setup.py:33]:
```suggestion
    'ploomber-core',
```
that's the right name of the package. I was surprised that this works as well! TIL pip replaces the `_` with `-`, that's why it didn't break

File: src/soopervisor/cli.py [hunk 2]
```diff
@@ -2,15 +2,19 @@
 import yaml
 import click
 
-from soopervisor import __version__
+from soopervisor import __version__ as ver
```
Comment (edublancas) [inline src/soopervisor/cli.py:5]:
```suggestion
from soopervisor import __version__
```

no need to rename here

File: src/soopervisor/cli.py [hunk 3]
```diff
@@ -2,15 +2,19 @@
...
-from ploomber.telemetry import telemetry
+from ploomber_core.telemetry.telemetry import Telemetry
+
+
+telemetry = Telemetry(api_key, ver, 'soopervisor')
 
 
 @click.group()
-@click.version_option(version=__version__)
+@click.version_option(version=ver)
```
Comment (edublancas) [inline src/soopervisor/cli.py:17]:
```suggestion
@click.version_option(version=__version__)
```
no need to rename

File: src/soopervisor/cli.py [hunk 4]
```diff
@@ -2,15 +2,19 @@
...
+from soopervisor import __version__ as ver
+from soopervisor import POSTHOG_API_KEY as api_key
 from soopervisor import config
 from soopervisor import exporter
 from soopervisor.enum import Backend, Mode
-from ploomber.telemetry import telemetry
+from ploomber_core.telemetry.telemetry import Telemetry
+
+
+telemetry = Telemetry(api_key, ver, 'soopervisor')
```
Comment (edublancas) [inline src/soopervisor/cli.py:13]: please create a `soopervisor.telemetry` module and instantiate this there. also move the posthog key to that file. same thing as we did with ploomber

**Review by edublancas — CHANGES_REQUESTED** [review]

File: src/soopervisor/cli.py (hunk 4 above)
Comment (Wxl19980214) [reply src/soopervisor/cli.py:13]: Ok

**Comment by Wxl19980214** [comment]
ready for merge

**Review by edublancas — APPROVED** [review]

**Comment by idomic** [comment]
@Wxl19980214 Nice job!

**Pull request merged**